cluster/examples/kubernetes/ceph/setup_clustermap.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+kubectl -n rook-ceph get pods --selector="app==rook-ceph-ganesha" --output=custom-columns=:.metadata.labels.instance,:.status.podIP | tail -n +2 |
+while read line; do
+	instance=$(echo $line | awk '{print $1}')
+	ip=$(echo $line | awk '{print $2}')
+	echo "$instance:$ip"
+	kubectl -n rook-ceph exec rook-ceph-tools -- rados -p nfs-ganesha setomapval clustermap $instance $ip
+done
```

Comment:
Oof, this patch got squashed in here when we pulled in one of my branches. This is a total hack that I was using to play around with migration. I think we should just drop the patch that adds this file.